pt-BR/mail/chrome/messenger/am-server-top.dtd

```diff
@@ -2,12 +2,12 @@
 
 <!ENTITY serverSettings.label "Receber mensagens">
 <!-- LOCALIZATION NOTE (popServer.label) : do not translate "POP" in below line -->
-<!ENTITY popServer.label "POP Mail Server">
+<!ENTITY popServer.label "Servidor de emails POP">
 <!-- LOCALIZATION NOTE (imapServer.label) : do not translate "IMAP" in below line -->
-<!ENTITY imapServer.label "IMAP Mail Server">
+<!ENTITY imapServer.label "Servidor de emails IMAP">
 <!-- LOCALIZATION NOTE (newsServer.label) : do not translate "NNTP" in below line -->
-<!ENTITY newsServer.label "News Server (NNTP)">
-<!ENTITY webmail.label "WebMail account">
+<!ENTITY newsServer.label "Servidor de Notícias (NNTP)">
+<!ENTITY webmail.label "Conta de emails">
```

Comment:
@brunotiussi Understood. After receiving Sherman's approval, either use e-mail or email (chose one to keep consistency).